activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -100,7 +100,7 @@ def test_indexes_logs_name
     end
 
     def test_table_exists_logs_name
-      @connection.table_exists?('items')
+      ActiveSupport::Deprecation.silence { @connection.table_exists?('items') }
```

Comment:
Should be the same behavior on `data_sources`. Can we rewrite that test to no longer use `tables`?